src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Abs.txt

```diff
@@ -76,4 +76,12 @@ false
 >> Abs("false")
 #Error(Kind=InvalidArgument)
 
+// ******** Date values and functions with dates ********
```

Comment:
This'll likely cause test failures when we bump the version for JS, as a heads up. 
Might want to either do that yourself, or keep a doc somewhere of the changed test results so it's easy to fix.

Reply:
Yes, I will bring the PFx version to JS when this is done, and handle the appropriate overrides.